service/common/src/main/java/org/apache/polaris/service/admin/PolarisAdminService.java

```diff
@@ -542,19 +544,17 @@ public PolarisEntity createCatalog(PolarisEntity entity) {
     PolarisAuthorizableOperation op = PolarisAuthorizableOperation.CREATE_CATALOG;
     authorizeBasicRootOperationOrThrow(op);
 
+    checkArgument(entity.getId() == -1, "Entity to be created must have no ID assigned");
```

Comment:
Do we really need to check argument ? Maybe we can simply ignore/override it ?

Reply:
Better give a HTTP/400 back, it's nice information